packages/ember/index.js

```diff
@@ -534,6 +535,7 @@ if (EMBER_GLIMMER_SET_COMPONENT_TEMPLATE) {
   Ember._setComponentTemplate = setComponentTemplate;
   Ember._templateOnlyComponent = templateOnlyComponent;
 }
+Ember._captureRenderTree = captureRenderTree;
```

Comment:
Can we have an official (intimate) module location for this also?

Reply:
What do I have to do to make it work? PR to the modules data repo?

Reply:
Ya, I can do it (and the related releases) just wanted to agree on what it actually will be.